bin/install-requirements.bash

```diff
@@ -7,9 +7,13 @@ export ANSIBLE_STDOUT_CALLBACK=yaml
 echo "This script will install all requirements"
 echo "Enter your password to start the install"
 
-if [ "${#}" -ge 2 ] && [ "${2}" = "--user" ]; then
+if [ "${#}" -ge 1 ] && [ "${1}" = "--user" ]; then
```

Comment:
This is incorrect if it's executed through `bin/ck8s`. If you want to keep this then you need to `shift` there as well.

Reply:
ah ye, that makes sense. Would a case statement with a loop make more sense here then maybe? Then both `bin/ck8s` and just running the script individually would work. So this script itself just checks for any arguments and if they are equal to the expected ones it does the right thing.

Reply:
yeah I guess, I went with just adding `shift` to `bin/ck8s` and didn't make a case statement. PTAL 😄